tests/fixtures/webextension_node_modules_bower/index.js

```diff
@@ -0,0 +1,5 @@
+function dummy(text, callback) {
+  callback(text;
```

Comment:
I skimmed this and didn't immediately see the syntax error, can you just add a comment explaining it? Thanks!